rescript/get_rdp_data.py

```diff
@@ -0,0 +1,133 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2022-2022, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+import tempfile
+import shutil
+import gzip
+# import warnings
+# import hashlib
+# import pandas as pd
+import qiime2
+from urllib.request import urlretrieve
+# from urllib.error import HTTPError
+# from q2_types.feature_data import MixedCaseDNAFASTAFormat, DNAFASTAFormat
+# from q2_feature_table import merge_seqs, merge_taxa
+
+
+def get_rdp_data(ctx,
+                 ref_db='Archaea+Bacteria',
```

Comment:
different parameter name? `ref_db` is ambiguous

Reply:
maybe call this something like "target"?

Reply:
and the options could then be "16S" and "26S"? (less to write and the markers are then clear from the name)